modules/core/src/test/resources/catalogs/landsat8/catalog-v1.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
what's the provenance of these? are they from the Landsat 8 catalog in AWS, or did you create them with stac4s / pystac? or something else?

Reply:
@jisantuc Landsat8 baked off from the https://landsat-stac.s3.amazonaws.com/landsat-8-l1/catalog.json and https://github.com/radiantearth/stac-spec/blob/master/item-spec/examples/landsat8-sample.json 

Also with removed unsupported fields (see Landsat8 assets) and taking into account some specifics of stac4s (taking into accound how it works with the label extension).

It is definitely not the 100% source for truth, but at least some close to real example.